### ISSUE TITLE
Add PurgeInstanceHistory webhook.

### ISSUE THREAD

**Issue by kashimiz** [Azure/azure-functions-durable-extension#543]
In order for the Durable Functions JavaScript shim to support the [PurgeInstanceHistoryAsync](https://azure.github.io/azure-functions-durable-extension/api/Microsoft.Azure.WebJobs.DurableOrchestrationClient.html#Microsoft_Azure_WebJobs_DurableOrchestrationClient_PurgeInstanceHistoryAsync_System_String_) methods, a two webhooks will need to be added to the extension:

**Purge Instance History by Instance ID**
```bash
GET /runtime/webhooks/durabletask/instances/{instanceId}/?taskHub={taskHub}&connection={connection}&code={systemKey}
```

**Purge Instance History with Filters**
```bash
GET /runtime/webhooks/durabletask/instances/?createdTimeFrom={createdTimeFrom}&createdTimeTo={createdTimeTo}&runtimeStatus={runtimeStatus,runtimeStatus,...}
```

This work is necessary to resolve Azure/azure-functions-durable-js#30.

**Comment by gled4er** [comment]
Hello @cgillum  @kashimiz,

I wanted to confirm that we should use **GET** HTTP verb for the purging history query. 

I think it can be misleading for the developers since most of the time the **GET** operations will only provide the current state of the system without causing any changes. On the other hand, here we are talking about purging of history so I think maybe it will be better to have **DELETE** or at least **POST** verb indicating that we are going to change the current state of the system.

Let me know if I am missing part of the context you have for this task.

Thank you!

**Comment by cgillum** [comment]
@gled4er I agree that it should not be **GET** since this operation has side effects. I think **DELETE** would be more appropriate in this case.

**Comment by gled4er** [comment]
Hello @cgillum,

Thank you very much for the confirmation!

**Comment by cgillum** [comment]
Resolved in [v1.8.0](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.DurableTask/1.8.0) release.
